README.md

```diff
@@ -1,2 +1,49 @@
 # lev-api-stress
-Stress testing for the `lev-api`.
+Stress testing for the [LEV API], based on the [Artillery.io] tool.
+
+
+## Running the test
+The performance tests can be started with the following command:
+```shell script
+./entrypoint.sh <path-to-config-file>
+```
+
+
+## Configuration
+There are two parts to the configuration:
+ - `./high-load.yml` - the main config file for [Artillery.io], containing:
+   - basic test setup
+   - security details
+   - config for the phases
+   - payload parameters (the information to match)
+   - scenarios (what requests to send, and how responses are processed)
+ - an extra file supplied to `entry-point.sh`, with security details:
+   - an endpoint for the target service
+   - an authorization token
+   - SSL certificate information
+   - files with search terms
+
+### A note about scenarios
+Each scenario defines a type of dummy user action which Artillery will mimic
+during the test. There two main actions; a search (by name and date of birth),
+and a direct lookup (by system number). Each of these actions has two different
+forms; a _probably_ successful form where the parameters are randomly selected
+from the corresponding payload file, and a _probably_ not successful form where
+parameters are randomly generated (see `./src/randomisor.js`).
```

Comment:
I wonder if it's worth mentioning why this is?

i.e.
1. Random searches that are likely to return no results.
2. Less random searches that are likely to return a result but run the risk of getting cached by the database reducing the validity of the test.

Reply:
Ah you do mention this below...